rollup.config.js

```diff
@@ -2,21 +2,68 @@ import babel from 'rollup-plugin-babel'
 import css from 'rollup-plugin-css-porter'
 import pkg from './package.json'
 
+const fs = require('fs')
+
+/**
+ * Custom rollup plug-in which bundles all transforms' CSS in a single file.
+ * @param  {callback} options Rollup plug-in options func
+ * @return {void}
+ */
+const wikimediaPageLibraryTransformCSSRollupPlugin = (options = {}) => {
```

Comment:
Ok was able to really simplify this. All I had to do was add a `sort-imports` eslint exception to index.js

I also was able to vastly simplify the custom rollup plugin and just use it instead of the `porter` plugin for both css files.